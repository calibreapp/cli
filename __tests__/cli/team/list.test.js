import {
  runCLI,
  setupIntegrationServer,
  teardownIntegrationServer
} from '../../utils'

import listTeams from '../../fixtures/listTeams.json'

describe('team list', () => {
  beforeAll(async () => await setupIntegrationServer(listTeams))
  afterAll(async () => await teardownIntegrationServer())

  test('lists all teams', async () => {
    const out = await runCLI({
      args: 'team list'
    })
    expect(out).toMatchSnapshot()
  })
})
