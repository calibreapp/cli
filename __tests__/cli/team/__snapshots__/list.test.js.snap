// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`team list lists all teams 1`] = `
"2 sites
SLUG        | NAME     | DESCRIPTION     
calibre     | Calibre  | Default team    
engineering | Monitors | Engineering team"
`;
